src/QueryType/CoreAdmin/Query/Action/AbstractAction.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Solarium\QueryType\CoreAdmin\Query\Action;
+
+use Solarium\Core\Configurable;
+
+/**
+ * CoreAdmin query command base class.
+ */
+abstract class AbstractAction extends Configurable
+{
+    /**
+     * Returns command type, for use in adapters.
+     *
+     * @return string
+     */
+    abstract public function getType();
```

Comment:
you added the typehint for the other methods. the `getType(): string;` is missing here

Reply:
thx, fixed that